app/controllers/change_requests_controller.rb

```diff
@@ -13,15 +13,26 @@ def create
   end
 
   def create_change_request
+    puts "#{params}"
+    puts "type: #{params[:type]}"
+    puts "parent: #{params[:parent_resource_id]}"
     if params[:resource_id]
       @change_request = ResourceChangeRequest.create(object_id: params[:resource_id], resource_id: params[:resource_id])
     elsif params[:service_id]
       @change_request = ServiceChangeRequest.create(object_id: params[:service_id], resource_id: Service.find(params[:service_id]).resource_id)
-    elsif params[:address_id]
-      @change_request = AddressChangeRequest.create(object_id: params[:address_id], resource_id: Address.find(params[:address_id]).resource_id)
+    elsif params[:address_id] || params[:type] == "addresses"
+      if params[:parent_resource_id]
```

Comment:
Shouldn't you invoke the create if the params[:type] is set and params[:address_id] is absent? Or, better, take advantage of the action type?

Reply:
@richardxia I'm happy to start using the `:action` param! One question I had was if I should also require / look for the `:action` param for adding a phone to a resource. This currently works very similarly to addresses.

Reply:
I think it'd be a good idea to look for it when adding resources, but let's not require it for now, and just default to `:edit`. That way, we have the flexibility of updating the frontend site at a later date, instead of requiring it to be updated right now.

Reply:
@richardxia I'm running into an issue where when I try to set the `:action` param I can't seem to override `create`: 
```
{  
    "action": "edit",
    "change_request": {
        "number": "(415) 555-5555",
        "service_type": "general inquiries"
    },
    "type": "phones",
    "parent_resource_id": 1
}
```
results in params of : 
```
{"action"=>"create", "change_request"=>{"number"=>"(415) 555-5555", "service_type"=>"general inquiries"}, "type"=>"phones", "parent_resource_id"=>1, "controller"=>"change_requests"}
```
Any ideas on what I'm missing here?